home-service/pom.xml

```diff
@@ -17,6 +17,21 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
```

Comment:
Why do we need jwt here?

Reply:
I added a JWT service for generating a token, but since we moved this into a separate task, I will remove it